From e0e0bd01f22daf80b922d152603e3ea422add828 Mon Sep 17 00:00:00 2001
From: Liu Gaoyong <liugaoyong_88@163.com>
Date: Wed, 23 Sep 2026 11:29:02 +0800
Subject: [PATCH] Update output directory and modify print_config

Changed default output directory from './zzz' to '.' and removed cleanup of output directory in print_config function.
---
 src/graphatoms/enterpoint/config/__init__.py | 5 +----
 1 file changed, 1 insertion(+), 4 deletions(-)

diff --git a/src/graphatoms/enterpoint/config/__init__.py b/src/graphatoms/enterpoint/config/__init__.py
index c0ae7cb..25e1f0a 100644
--- a/src/graphatoms/enterpoint/config/__init__.py
+++ b/src/graphatoms/enterpoint/config/__init__.py
@@ -66,7 +66,7 @@ class Config:
 
     calculator: CalcConfig = MISSING
 
-    outputs: str = "./zzz"
+    outputs: str = "."
     max_steps: int = 100000
     max_times: float = 1000.0  # seconds
     run_type: str = "otfkmc"  # otfkmc, rxngen
@@ -98,8 +98,6 @@ class Config:
     version_base=None,
 )
 def print_config(cfg: Config) -> None:
-    from shutil import rmtree
-
     print(OmegaConf.to_yaml(cfg))
     print("""
 hydra:
@@ -107,4 +105,3 @@ def print_config(cfg: Config) -> None:
     dir: ${outputs}
   verbose: ${debug}
 """)
-    rmtree(Path(cfg.outputs))
